Networking/NetworkingTests/Remote/WCPayRemoteTests.swift

```diff
@@ -645,23 +604,6 @@ final class WCPayRemoteTests: XCTestCase {
         XCTAssertEqual(location.locationID, expectedLocationID)
     }
 
-    func test_loadDefaultReaderLocation_properly_handles_error_response() throws {
-        let remote = WCPayRemote(network: network)
-
-        network.simulateResponse(
-            requestUrlSuffix: "payments/terminal/locations/store",
-            filename: "wcpay-location-error"
-        )
-
-        let result: Result<Customer, Error> = waitFor { promise in
-            remote.fetchOrderCustomer(for: self.sampleSiteID, orderID: self.sampleOrderID) { result in
```

Comment:
this test case doesn't seem to be testing what the test name claims 🤔 I'm guessing if this was meant to test `loadDefaultReaderLocation` function like in `test_loadDefaultReaderLocation_properly_returns_location` above. Maybe this test case can be restored and this line can call `loadDefaultReaderLocation` to see if this passes?